lib/index.js

```diff
@@ -219,25 +219,12 @@ class WatchDetector {
 
       debug.info('detected watchman: %s', version);
 
-      if (semver.satisfies(version, '>= 3.0.0')) {
-        debug.info('watchman %s does satisfy: %s', version, '>= 3.0.0');
+      result.watcher = WATCHMAN;
```

Comment:
Instead of removing all semver checks and assuming all timestamp versions work, shouldn't we instead widen the checks to additionally include timestamp based checks?

Reply:
How could we do this?

Reply:
Parsing the timestamp version pattern and ensuring it's at a minimum version? This may not be necessary, if the cutover to using timestamp version numbers _always_ includes the minimum version. For instance, if we're simply checking for old versions of watchman, and all timestamp versions are exempt from this since they're already higher than the associated required equivalent semver version, we may not even need to do this.

Reply:
@scalvert - I was thinking that at this point we can probably assume we are never going to see such an old watchman that we care about (e.g. < 3.0.0). 

If you take a look at the timestamps (https://github.com/facebook/watchman/releases/tag/v3.0.0) 3.0.0 was released in 2014. I bet we don't even have users on machines in common use are that many years old. 😝

Reply:
Ya this is what Gabe and I were discussing offline. It seems that it's not even really possible to install a semver version of watchman - only timestamp versions.

This makes sense in that case.